### ISSUE TITLE
Coupled 1D and 3D model 

### ISSUE THREAD

**Issue by adriennekscott** [gibbonCode/GIBBON#186]
@Kevin-Mattheus-Moerman Can we alternate calls between an FEBio 3D model and a Matlab generated 1D LPN within a single time step using gibbon?

**Comment by Kevin-Mattheus-Moerman** [comment]
Yes potentially. Not sure what the 1D model is but you start/pause/alter/resume FEBio jobs. I have no demo for this but I think it is mainly an FEBio question. If FEBio allows what you want then we can code it in GIBBON.

**Comment by Kevin-Mattheus-Moerman** [comment]
Closed as this issue is likely stale.
